creator-node/src/fileManager.js

```diff
@@ -376,6 +376,17 @@ function getFileExtension (fileName) {
   return (fileName.lastIndexOf('.') >= 0) ? fileName.substr(fileName.lastIndexOf('.')).toLowerCase() : ''
 }
 
+function hasEnoughStorageSpace ({ storagePathSize, storagePathUsed, maxStorageUsedPercent }) {
```

Comment:
can you add a short comment pls
i know not all functions in this file are commented (which is my bad) but lets fix moving fwd